src/beloved/feature.js

```diff
@@ -1 +1,57 @@
 console.log ("Habeeb");
+
+//14 ----------- number
+//-80 ----------- number
+//8.5 ----------- number
+//true ----------- boolean
+//(5 < 17) ---------- function
+//null ------------ undefined
+//3.142 ------------ number
+//"pi" ------------ string
+//67 + 18 ----------- number
+
+//question
+//write a program to produce binary values from decimals
+
+var binaryConverter = function(number){
+        var reverseBinary = [];
+        do{
+                reverseBinary.push(number%2);
+                number = Math.floor(number/2);
+                console.log(number);
+        }while(number>=1)
+        return console.log(reverseBinary.reverse().join(''));
```

Comment:
You do not need to log the result to the console. The `return` statement is adequate.